composer.json

```diff
@@ -7,40 +7,40 @@
     "require": {
         "php": "^7.3 || ^8.0",
         "silverstripe/recipe-plugin": "^1",
-        "silverstripe/installer": "4.x-dev",
-        "silverstripe/recipe-authoring-tools": "1.x-dev",
-        "silverstripe/recipe-blog": "1.x-dev",
-        "silverstripe/recipe-collaboration": "1.x-dev",
-        "silverstripe/recipe-form-building": "1.x-dev",
-        "silverstripe/recipe-reporting-tools": "1.x-dev",
-        "silverstripe/recipe-services": "1.x-dev",
-        "silverstripe/recipe-content-blocks": "2.x-dev",
-        "silverstripe/recipe-solr-search": "2.x-dev",
+        "silverstripe/installer": "4.10.x-dev",
+        "silverstripe/recipe-authoring-tools": "1.10.x-dev",
+        "silverstripe/recipe-blog": "1.10.x-dev",
+        "silverstripe/recipe-collaboration": "1.10.x-dev",
+        "silverstripe/recipe-form-building": "1.10.x-dev",
+        "silverstripe/recipe-reporting-tools": "1.10.x-dev",
+        "silverstripe/recipe-services": "1.10.x-dev",
+        "silverstripe/recipe-content-blocks": "2.10.x-dev",
+        "silverstripe/recipe-solr-search": "2.10.x-dev",
```

Comment:
We need to update the matching branch on silverstripe/recipe-solr-search

Reply:
2.10 exists https://github.com/silverstripe/recipe-solr-search/tree/2.10

Reply:
It exists but it needs a PR to update its composer file

Reply:
Created https://github.com/silverstripe/recipe-solr-search/pull/3